src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -27,7 +27,11 @@ import { OptInExampleFlyout } from './opt_in_example_flyout';
 
 type TelemetryService = TelemetryPluginSetup['telemetryService'];
 
-const SEARCH_TERMS: string[] = [
+/**
+ * These are the terms provided to Advanced Settings that map to this section. When searching,
+ * Advanced Settings will match against these terms to show or hide the section.
+ */
```

Comment:
Thanks for adding this comment, really useful!

nit: do you think we could move this to a file `constants.ts` and place it at the root of this plugins public dir?

Reply:
Sure, I'm going to clean up the i18n, so I can make this change, too.